### DISCUSSION TITLE
Dbeaver option for Importing into Volatile Table using Fastload in Teradata?

### DISCUSSION THREAD

**Discussion by anthonykamarflores** [dbeaver/dbeaver#21179] [Q&A]
Hi,

I am new to using Dbeaver. I am using DBeaver with a Teradata connection. Typically, you can import data into a volatile table via fastload in Teradata Studio, however, since I switched over to DBeaver recently, I don't see an obvious option to import data into a volatile table. It seems like the only way to import data into a table in DBeaver is via the Database Navigator: right clicking on the Database Navigator and then clicking "Import Data" will guide the user to the import wizard. However, Volatile (a.k.a temp) tables will not appear in this wizard, because they do not exist in the Schema. These tables only exist in the spool of the user and are then purged at the end of a session. So this option isn't viable.

Is there something I'm missing? Is there a way to utilize Fastload in Teradata with DBeaver for Volatile tables? Normal Importing wouldn't work in this case and neither does SQL statements that you can utilize in Teradata SQL Assistant (now deprecated by Teradata).

**Comment by LonwoLonwo** [comment]
Hello @anthonykamarflores 

Fastload is not implemented in DBeaver, yes, because there are too many restrictions to this type of data transfer.

Could you please explain to me - what is the specific reason for using volatile and not ordinary tables for data import?

**Reply by anthonykamarflores** [reply]
That's understandable. 

I am not given access to create ordinary tables in the environment I am working in except in a very limited schema. Right now I am utilizing some workarounds that is good for the short term - but it would be best to be able to use volatile tables for short term analysis, ad-hoc reporting, etc.

Is this not a possibility with DBeaver, even without Fastload?

**Reply by LonwoLonwo** [reply]
DBeaver uses the Teradata driver to return the tables list. I do not see volatiles tables in this list. Even `select * from dbc.Tables` doesn't return me a volatile table. So, I'm not sure what we can do here...
